src/Bicep.Cli.E2eTests/tests/build.test.js

```diff
@@ -0,0 +1,25 @@
+const path = require("path");
+const fs = require("fs");
+const runBicepCommand = require("../command");
+
+const exampleDirectory = path.resolve(__dirname, "../examples/101/aks/");
+const exampleBicepFile = path.join(exampleDirectory, "main.bicep");
+const exampleJsonFile = path.join(exampleDirectory, "main.json");
+
+describe("bicep build", () => {
```

Comment:
>describe [](start = 0, length = 8)

We should also do a failed build test to make sure we're logging warnings and errors to stderr.

Reply:
Good pt. Added a test.